### DISCUSSION TITLE
Use of aggregate functions is not allowed

### DISCUSSION THREAD

**Discussion by MrVibe** [supabase/supabase#28112] [Questions] [answered]
Using latest supbase version, running a very simple query.

const stats = await supabaseAdmin
    .from(`stats`)
    .select('emails.sum()');

and getting the error :

'Use of aggregate functions is not allowed'

it seems this is still disabled in open source version or maybe I am missing something.

**Comment by GaryAustin1** [comment, marked as answer]
You have to enable them.
https://supabase.com/blog/postgrest-aggregate-functions#staying-safe-with-aggregate-functions
